api/src/test/java/org/openmrs/module/m2sysbiometrics/service/RegistrationServiceTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldTryToRegisterNationallyWithoutException() throws Exception {
         when(nationalBioServerClient.enroll(subject.getSubjectId(), capture.getTemplateData())).thenReturn(EXISTING_RESULT_XML);
 
         //when
-        registrationService.registerNationally(subject, capture);
+        registrationService.registerNationally(subject.getSubjectId(), capture);
```

Comment:
I would define a new national guid to emphasize that this it is not related to subjectid

Reply:
Ok I will, got a bit lazy in this test